packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -105,7 +105,11 @@ export const defaultMyChatMessageContainer: ComponentSlotStyle = {
 export const defaultChatMessageContainer: ComponentSlotStyle = {
   maxWidth: '100%',
   minWidth: `${CHAT_MESSAGE_CONTAINER_MIN_WIDTH_REM}rem`,
-  marginRight: '0rem'
+  marginRight: '1rem',
+  ' img': {
```

Comment:
`' img` -> '> img' might be better as it will apply styles to all img tags recursively

Reply:
Thanks Anjul - I got you concerns about this, this is the intention we want to add it recursively. Because teams sometimes will generate nested html structure so we can't only go one level deep to its children